streaming/base/_constant.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
why bother prefixing filename with _? maybe call it constants.py?

Reply:
Kept it _ because it is not being used externally by the user, however, changed it back to `constants.py` file to make it uniform with other files.